corehq/apps/reports/standard/monitoring.py

```diff
@@ -471,7 +471,7 @@ def all_users(self):
         return sorted(users, key=lambda u: u['username_in_report'])
 
     def paginate_list(self, data_list):
-        if self.pagination:
```

Comment:
I'm thinking we want to make the change at the ```GenericTabularReport``` level, since I've observed this in other reports as well - for instance check out https://staging.commcarehq.org/hq/sms/sms_billables/?date_sent_startdate=2015-02-14&date_sent_enddate=2015-02-20&date_created_startdate=2015-02-14&date_created_enddate=2015-02-20&show_billables=&domain=